tests/acceptance/features/webUIPublicShareLink/publicShareLinkExpirationPolicy.feature

```diff
@@ -0,0 +1,66 @@
+@webUI @insulated @disablePreviews
```

Comment:
Everything seems great, one little suggestion: IMO feature file can be named `publicLinkShareExpirationPolicy.feature` rather _publicShareLink..._

Reply:
The test suite and all its feature files have `publicLinkShare` - we can change that in a separate refactoring PR